sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -213,6 +213,29 @@ from azure.core.credentials import AzureKeyCredential
 from azure.ai.documentintelligence import DocumentIntelligenceClient
 from azure.ai.documentintelligence.models import AnalyzeResult
 
+def get_words(page, line):
+    result = []
+    for word in page.words:
+        if _in_span(word, line.spans):
+            result.append(word)
+    return result
+
+def _in_span(word, spans):
```

Comment:
I think we should either append an `_` to all of the helper functions or remove this one since it's just another helper like the others.

Reply:
Appended "_" to all helper functions which defined inside.